commands/mentorship.go

```diff
@@ -0,0 +1,28 @@
+package commands
+
+import (
+	"bytes"
+
+	mentors "github.com/CaravanaCloud/bino/mentors"
+)
+
+var mentorshipCommand = command{
+	CanRun: canRun,
+	Run:    run,
+}
+
+var Mentorship = CommandRunner{
+	Command: mentorshipCommand,
+}
+
+func canRun(message string) bool {
+	return message == "mentorias"
+}
+
+func run(message string) (string, error) {
+	var stringBuffer bytes.Buffer
```

Comment:
here, you can use this [named return values](https://tour.golang.org/basics/7)

Reply:
you can use `fmt.Sprintf()` to interpolation variables inside strings and return string.

Reply:
Fixed!

This is interesting, when do you think we should be using named returns? Honestly I did not even remember it was a thing :P